src/dataflow/analysis/particle-node.ts

```diff
@@ -107,8 +111,10 @@ export class ParticleOutput implements Edge {
     this.connectionSpec = connection.spec;
     this.label = `${particleNode.name}.${this.connectionName}`;
     
-    const claim = particleNode.claims.get(this.connectionName);
-    this.claim = claim ? claim.expression : null;
+    const claim = particleNode.claims.find(claim => {
+      return (this.connectionName === claim.handle.name);
+      }, this);
```

Comment:
Do you actually need to supply "this" here? I think this works:

`particleNode.claims.find(claim => this.connectionName === claim.handle.name)`

Reply:
But also, I think we can just use `connection.handle.claims` like you did in ParticleInput above, no?

Reply:
The JS array documentation for find() says that if you don't supply a "this", it's undefined, so if I use it, I supply it. That said, I tried it without the "this" and it works, so I'll leave it off.
As for taking the claims from the handle, I would have thought so, but if I do that several tests fail. It appears that this code is being called before the handle connections are fully set up with claims.